projects/plugins/boost/app/lib/class-storage-post-type.php

```diff
@@ -142,7 +142,7 @@ public function get( $key, $default ) {
 			return $default;
 		}
 
-		wp_cache_set( $key, $value['data'], $this->post_type_slug(), HOUR_IN_SECONDS );
+		set_transient( $this->post_type_slug() . '_' . $key, $value['data'], HOUR_IN_SECONDS );
```

Comment:
At first I was wondering if we should worry about the length of that transient name, but I doubt we'll ever have a post type slug that's more than 172 (transient name limit) - 1 (the underscore) - 45 (`$cache_key`) = 126 characters long :)